manifests/d/DrewNaylor/guinget/0.3/DrewNaylor.guinget.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
+PackageIdentifier: DrewNaylor.guinget
+Publisher: DrewNaylor
```

Comment:
```suggestion
Publisher: Drew Naylor
```

Reply:
I prefer my name to be one word because it's consistent with my GitHub profile and it makes it more like a company and less like a person's name. Thinking about it, maybe the Start menu folder name my programs show up in should be changed to not have a space for consistency.

Reply:
`Drew Naylor` is the Publisher you've set which appears in Windows which is why this NEEDS to be `Drew Naylor` instead of `DrewNaylor` so there is no issues.

Reply:
Oh, didn't know it had to be the same thing. I'll get rid of the space and update the manifest with a new installer. This may take a few minutes.

Reply:
`Publisher`, `PackageName` and `PackageVersion` needs to match what Windows sees otherwise `winget upgrade` and/or `winget list` will bug out, i see i was the one who reviewed your last PR and didn't quality check it but instead broke it🤦‍♂️

Reply:
Figured the version and package name had to match, but didn't think the publisher had to as well. Makes sense it would break upgrades and stuff.